### ISSUE TITLE
Theory-Inspired Path-Regularized Differential Network Architecture Search

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#128]
# TL;DR
DARTS, one of the Neural Architecture Searches methods, often degrades accuracy because it select often skipping connections, but they proved theoretically that this is because the convergence speed is faster than other operations such as Conv. Furthermore, they proposed a PR-DARTS that prevent it and confirmed that the accuracy of the PR-DARTS was improved.

![skipcor](https://user-images.githubusercontent.com/24524018/95158727-4f04a680-07d7-11eb-950c-79336a5c6d57.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.16537

# Submission Dates(yyyy/mm/dd)
2020/06/30

# Authors and institutions
Pan Zhou Caiming Xiong Richard Socher Steven C.H. Hoi
- Salesforce Research

# Methods

# Results

# Comments
